composer.json

```diff
@@ -20,8 +20,7 @@
         "cakephp/cakephp": "~3.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "4.1.*",
-        "cakephp/cakephp-codesniffer": "dev-master"
```

Comment:
They are part of the travis setup, and usually installed globally on dev machines. No?